public-api/src/integration/java/org/pmiops/workbench/publicapi/DataBrowserControllerIntegrationTest.java

```diff
@@ -42,7 +46,18 @@ DataBrowserApi client() {
       if (Strings.isNullOrEmpty(basePath)) {
         throw new RuntimeException("Required env var '" + DB_API_BASE_PATH + "' not defined");
       }
-      api.setApiClient(new ApiClient().setBasePath(basePath));
+      ApiClient apiClient = new ApiClient();
+      apiClient.setBasePath(basePath);
+      if (basePath.contains("aou-db-staging")) {
+        String token = "";
+        try {
+          BuildIapRequest buildRequest = new BuildIapRequest();
+          token = buildRequest.buildIapRequest(CLIENT_ID);
+        } catch (IOException ie) {
```

Comment:
should probably just let this go uncaught so the test fails

Reply:
I cannot not catch it. I should either catch it or throw it.

Reply:
You should be able to simply change the method signature to `DataBrowserApi client() throws IOException {`

Reply:
Throwing from client has thrown error previously for a different reason may be. That whole branch was messed up. It works fine now. Thanks.